.github/workflows/test.yml

```diff
@@ -10,35 +10,35 @@ on:
       # - `pull_request` see below
       # - https://github.com/prisma/prisma/blob/master/src/.buildkite/test/buildkite-entry.sh
       # - https://github.com/prisma/prisma/blob/master/src/.buildkite/publish/buildkite-entry.sh
-      - '*.md'
-      - '.vscode/**'
-      - 'docs/**'
-      - 'examples/**'
-      - 'LICENSE'
-      - 'src/.buildkite/**'
-      - 'src/.dockerignore'
-      - '*.bench.ts'
-      - 'src/scripts/ci/publish.ts'
-      - 'src/graphs/**'
+      - "*.md"
+      - ".vscode/**"
+      - "docs/**"
+      - "examples/**"
+      - "LICENSE"
+      - "src/.buildkite/**"
+      - "src/.dockerignore"
+      - "*.bench.ts"
+      - "src/scripts/ci/publish.ts"
+      - "src/graphs/**"
```

Comment:
Not sure which plugin/setting is making PRs look like this for this file, I think that my config will probably want to revert the change 😢

Reply:
Ahh bugger let me sort that out